docs/specviz/import_data.rst

```diff
@@ -2,5 +2,57 @@
 Import Data
 ***********
 
-Words go here
+There are two primary ways in which a user can load their data into the Specviz application. It should be noted that currently, Specviz only supports data that can be parsed as :class:`~specutils.Spectrum1D` objects.
 
+.. seealso::
+
+    `Reading from a File <https://specutils.readthedocs.io/en/stable/spectrum1d.html#reading-from-a-file>`_
+        SpecUtils documentation on loading data as :class:`~specutils.Spectrum1D` objects.
+
+Importing data through the GUI
+------------------------------
+
+The first way that users can parse their data into the Specviz application is
+by using the "Import Data" button in the application's user interface.
+
+.. image:: img/specviz_viewer.png
+
+This process is fairly straightforward, users need only click on the "Import Data" button:
+
+.. image:: img/import_data_1.png
+
+and enter the path of file that can be parsed as a :class:`~specutils.Spectrum1D` in the text field:
+
+.. image:: img/import_data_2.png
+
+After having clicked "Import", the data file will be parsed and loaded into the application. A notification will appear to let users know if the data importing was successful. Afterward, the new data set can be found in the "Data" tab of each viewer's options menu.
```

Comment:
After clicking "Import"
and
to let users know if the data import was successful.